packages/ui/src/components/va-checkbox/VaCheckbox.vue

```diff
@@ -124,18 +124,25 @@ export default defineComponent({
       'va-checkbox--on-keyboard-focus': hasKeyboardFocus.value,
     }))
 
+    const getPaddingStyle = () => {
+      switch (true) {
+        case !props.label:
+          return ''
+        case props.vertical:
+          return 'var(--va-checkbox-vertical-padding)'
+        case Boolean(props.arrayValue):
```

Comment:
There were one issue, we need to convert arrayValue to boolean, otherwise switch will compare true and "One" which is false.